auth-web/src/views/auth/staff/StaffDashboardView.vue

```diff
@@ -226,17 +226,54 @@
         <SafeEmailView />
       </template>
     </BaseVExpansionPanel>
+
+    <!-- Involuntary Dissolution -->
+    <v-card
+      v-if="showInvoluntaryDissolutionTile"
+      flat
+      class="mt-4 pa-8"
+    >
+      <v-row
+        align="center"
+        justify="space-between"
+      >
+        <v-col>
+          <header>
+            <h2 class="mb-0">
+              Involuntary Dissolution
+            </h2>
+            <p class="mt-3 mb-0">
+              Set up and manage automation for Involuntary Dissolution of businesses
```

Comment:
Could you double check with UX the wording here?
here is what I found on current design:
![image](https://github.com/bcgov/sbc-auth/assets/16507223/0e48cb31-971a-45f9-a9f1-2fe91da03b1e)

Reply:
Good catch Jia! Yes, you're right but the wording has changed. Andy has made changes to the design:
![andy changes](https://github.com/bcgov/sbc-auth/assets/122301442/b32d9987-b52f-47f4-b225-1cf4f1ef70d1)

So, I had the text the same as his latest changes.